packages/cadl-ranch-specs/http/payload/xml/main.tsp

```diff
@@ -0,0 +1,195 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@doc("Sends and receives bodies in XML format.")
+@scenarioService("/payload/xml")
+namespace Payload.Xml;
+
+@doc("Contains fields of primitive types.")
+model SimpleModel {
+  name: string;
+  age: int32;
+}
+
+@doc("Contains fields of arrays of primitive types.")
+model ModelWithSimpleArrays {
+  colors: string[];
+  counts: int32[];
+}
+
+@doc("Contains an array of models.")
+model ModelWithArrayOfModel {
+  items: SimpleModel[];
+}
+
+@doc("Contains an optional field.")
+model ModelWithOptionalField {
+  item: string;
+  counts?: int32;
+}
+
+@scenario
+@route("/simpleModel")
+@scenarioDoc("""
+Returns an instance of SimpleModel.
+Expected response body:
+```xml
+<SimpleModel>
+  <Name>foo</Name>
+  <Age>123</Age>
+</SimpleModel>
+```
+""")
+@get
+op getSimpleModel(): {
```

Comment:
Nit but I would create a structure where we have 

interface SimpleModel {
  get()
  put();
}

make it nicer in the dashboard, you can then also use template to easily define both as I assume the get and put should send/receive teh same data

Reply:
there is a lot of other spec with this `GetOrSend` tempalte interface that is reused for example

Reply:
fixed